BatchProcessing/batch.camel.yaml

```
@@ -0,0 +1,52 @@
+- route:
+    id: route-3731
+    from:
+      id: from-6471
+      uri: file:C:\Users\ajdesai\Demo\kaoto\
```

Would it be possible to use a folder relative to this file, perhaps something like this: 
```suggestion
      uri: file:./samples/
```
so we could run your example directly from [Dev Spaces](https://developers.redhat.com/products/openshift-dev-spaces/overview)?